interop/xds/client/client.go

```diff
@@ -84,10 +84,24 @@ type accumulatedStats struct {
 	numRpcsFailedByMethod    map[string]int32
 }
 
+// copyStatsMap makes a copy of the map, and also replaces the RPC type string
+// to the proto string. E.g. "UnaryCall" -> "UNARY_CALL".
 func copyStatsMap(originalMap map[string]int32) (newMap map[string]int32) {
 	newMap = make(map[string]int32)
 	for k, v := range originalMap {
-		newMap[k] = v
+		var kk string
+		switch k {
+		case unaryCall:
```

Comment:
Can we not define unaryCall and emptyCall as UNARY_CALL/EMPTY_CALL to work around this instead?

Reply:
No, the same consts are also used to parse the flags `"EmptyCall:key1:value1"`
```
	rpcMetadata     = flag.String("metadata", "", "The metadata to send with RPC, in format EmptyCall:key1:value1,UnaryCall:key2:value2")
```